### DISCUSSION TITLE
How to force reconfigure ?

### DISCUSSION THREAD

**Discussion by Stoufiler** [Thomas55555/husqvarna_automower#312] [Q&A]
Hi everyone !

I wanted to know how can I force reconfigure ?
I need to change my credentials to set-up `client_id` and `client_secret`

<img width="886" alt="image" src="https://user-images.githubusercontent.com/44873931/179249440-1f7f4d2c-8a07-4f3e-8da7-b8ebccf40974.png">

Thanks a lot

**Comment by prairiesnpr** [comment]
I think you can just goto settings - > integrations - > three dot menu - > application credentials - add application credential

Select Husqvarna Automower for the integration and fill out the form.

**Reply by Stoufiler** [reply]
Thanks a lot it worked !
